internal/command/test_cleanup.go

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package command
+
+import (
+	"context"
+	"strings"
+	"time"
+
+	backendInit "github.com/hashicorp/terraform/internal/backend/init"
+	"github.com/hashicorp/terraform/internal/backend/local"
+	"github.com/hashicorp/terraform/internal/command/junit"
+	"github.com/hashicorp/terraform/internal/logging"
+	"github.com/hashicorp/terraform/internal/moduletest"
+	"github.com/hashicorp/terraform/internal/tfdiags"
+)
+
+// TestCleanupCommand is a command that cleans up left-over resources created
+// during Terraform test runs. It basically runs the test command in cleanup mode.
+type TestCleanupCommand struct {
+	Meta
+}
+
+func (c *TestCleanupCommand) Help() string {
+	helpText := `
+Usage: terraform [global options] test cleanup [options]
+
+  Cleans up left-over resources created during Terraform test runs.
+
+  By default, this command ignores the skip_cleanup attributes in the manifest
+  file. Use the -repair flag to override this behavior. Additionally, the
+  -target flag allows specifying which state files to clean up.
+
+Options:
+
+  -repair               Overrides the skip_cleanup attribute in the manifest
+                        file and attempts to clean up all resources.
+
+  -target=statefile     Specifies the state file(s) to clean up. Use this option
```

Comment:
this hasn't been implemented yet right? perhaps, remove mention of it until then?